fhir-model/src/test/java/com/ibm/fhir/model/util/test/ReferenceMappingVisitorTest.java

```diff
@@ -70,45 +69,211 @@ public void setUp() throws Exception {
                         .family(String.of("Doe"))
                         .build();
 
-        java.lang.String uUID = UUID.randomUUID().toString();
-
-        localRefMap = new HashMap<java.lang.String, java.lang.String>();
-        localRefMap.put("urn:uuid:" + uUID, "urn:romote:" + uUID);
-
-        Reference providerRef = Reference.builder()
-                        .reference(String.of("urn:uuid:" + uUID))
-                        .build();
-
-        patient = Patient.builder()
+        basePatient = Patient.builder()
                         .id(id)
                         .active(Boolean.TRUE)
                         .multipleBirth(Integer.of(2))
                         .meta(meta)
                         .name(name)
                         .birthDate(Date.of(LocalDate.now()))
-                        .generalPractitioner(providerRef)
                         .build();
     }
 
     @Test(enabled = true)
```

Comment:
we could probably drop the `enabled = true` instead of extending that pattern in the new tests (unless we think its specifically useful to toggle these ones)

Reply:
Done.